src/invoices/PrintVoucher/PrintContent/PrintContent.css

```diff
@@ -4,7 +4,7 @@
 
 @media print {
   @page {
-    size: A4 landscape;
+    size: A4;
```

Comment:
Actually, I'm not sure it this works. In mine chrome previous manual choice is selected always